---
layout: default
title: Studies
group: "in_local_navigation"
order: 5
---


## List of studies included in the eQTL Catalogue

### RNA-seq studies

| Study | Cell types or tissues | Conditions | Samples | Donors | Expression Atlas | Raw expression data | Raw genotype data | QC report |
|---|---|---|---|---|---|---|---|---|
[Alasoo_2018](http://dx.doi.org/10.1038/s41588-018-0046-7) | macrophages | IFNg, Salmonella, IFNg + Salmonella | 336 | 84 | [E-ENAD-41](https://www.ebi.ac.uk/gxa/experiments/E-ENAD-41) | [EGAD00001003204](https://www.ebi.ac.uk/ega/datasets/EGAD00001003204); [PRJEB18997](https://www.ebi.ac.uk/ena/data/view/PRJEB18997) | [EGAD00010001147](https://www.ebi.ac.uk/ega/datasets/EGAD00010001147); [PRJEB11752](https://www.ebi.ac.uk/ena/data/view/PRJEB11752) | [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/Alasoo_2018_QC_report.html)
[BLUEPRINT](https://doi.org/10.1016/j.cell.2016.10.026) | monocytes, neutrophils, CD4+ T cells |  | 554 | 197 | [E-ENAD-34](https://www.ebi.ac.uk/gxa/experiments/E-ENAD-34/) | [EGAD00001002671](https://www.ebi.ac.uk/ega/datasets/EGAD00001002671); [EGAD00001002674](https://www.ebi.ac.uk/ega/datasets/EGAD00001002674); [EGAD00001002675](https://www.ebi.ac.uk/ega/datasets/EGAD00001002675) | [EGAD00001002663](https://www.ebi.ac.uk/ega/datasets/EGAD00001002663) | [1]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/BLUEPRINT_SE_QC_report.html),[2]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/BLUEPRINT_PE_QC_report.html)
[GENCORD](https://doi.org/10.7554/eLife.00523) | LCLs[^2], fibroblasts, T cells |  | 560 | 195 | | [EGAD00001000425](https://www.ebi.ac.uk/ega/datasets/EGAD00001000425) | [EGAD00001000428](https://www.ebi.ac.uk/ega/datasets/EGAD00001000428) | [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/GENCORD_QC_report.html)
[GEUVADIS](https://doi.org/10.1038/nature12531) | LCLs[^2] |  | 445 | 445 | [E-GEUV-1](https://www.ebi.ac.uk/gxa/experiments/E-GEUV-1) | [E-GEUV-1](https://www.ebi.ac.uk/arrayexpress/experiments/E-GEUV-1) | 1000 Genomes | [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/GEUVADIS_QC_report.html)
[HipSci](https://doi.org/10.1038/nature22403) | iPSCs[^1] |  | 322 | 322 | [E-MTAB-4748](https://www.ebi.ac.uk/gxa/experiments/E-MTAB-4748) [E-ENAD-35](https://www.ebi.ac.uk/gxa/experiments/E-ENAD-35) | [EGAD00001003529](https://www.ebi.ac.uk/ega/datasets/EGAD00001003529); [PRJEB7388](https://www.ebi.ac.uk/ena/data/view/PRJEB7388) | [EGAD00010001147](https://www.ebi.ac.uk/ega/datasets/EGAD00010001147); [PRJEB11752](https://www.ebi.ac.uk/ena/data/view/PRJEB11752) | [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/HipSci_QC_report.html)
[Nedelec_2016](http://dx.doi.org/10.1016/j.cell.2016.09.025) | macrophages | Listeria, Salmonella | 493 | 168 |  | [GSE81046](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81046) | Not deposited | [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/Nedelec_2016_QC_report.html)
[Quach_2016](http://dx.doi.org/10.1016/j.cell.2016.09.024) | monocytes | LPS, Pam3CSK4, R848, IAV | 969 | 200 |  | [EGAD00001002714](https://www.ebi.ac.uk/ega/datasets/EGAD00001002714) | [EGAD00010001131](https://www.ebi.ac.uk/ega/datasets/EGAD00010001131) | [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/Quach_2016_QC_report.html)
[Schwartzentruber_2018](http://dx.doi.org/10.1038/s41588-017-0005-8) | sensory neurons |  | 98 | 98 | [E-ENAD-33](https://www.ebi.ac.uk/gxa/experiments/E-ENAD-33) | [EGAD00001003145](https://www.ebi.ac.uk/ega/datasets/EGAD00001003145); [PRJEB18630](https://www.ebi.ac.uk/ena/data/view/PRJEB18630) | [EGAD00010001147](https://www.ebi.ac.uk/ega/datasets/EGAD00010001147); [PRJEB11752](https://www.ebi.ac.uk/ena/data/view/PRJEB11752) | [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/Schwartzentruber_2018_QC_report.html)
[TwinsUK](http://dx.doi.org/10.1038/ng.3162) | adipose, LCLs[^2], skin, blood |  | 1364 | 433 |  | [EGAD00001001086](https://www.ebi.ac.uk/ega/datasets/EGAD00001001086); [EGAD00001001087](https://www.ebi.ac.uk/ega/datasets/EGAD00001001087); [EGAD00001001088](https://www.ebi.ac.uk/ega/datasets/EGAD00001001088); [EGAD00001001089](https://www.ebi.ac.uk/ega/datasets/EGAD00001001089) | Not deposited | [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/TwinsUK_QC_report.html)
[van_de_Bunt_2015](https://doi.org/10.1371/journal.pgen.1005694) | pancreatic islets |  | 117 | 117 | [E-ENAD-42](https://www.ebi.ac.uk/gxa/experiments/E-ENAD-42) | [EGAD00001001601](https://www.ebi.ac.uk/ega/datasets/EGAD00001001601) | [EGAD00001001601](https://www.ebi.ac.uk/ega/datasets/EGAD00001001601) | [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/van_de_Bunt_2015_QC_report.html)
[Schmiedel_2018](https://doi.org/10.1016/j.cell.2018.10.022) | 15 immune cell types | αCD3+αCD28 (4h) | 1331 | 91 |  | [phs001703.v1.p1](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001703.v1.p1) | [phs001703.v1.p1](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001703.v1.p1)| [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/Schmiedel_2018_QC_report.html)
[BrainSeq](https://doi.org/10.1038/s41593-018-0197-y) | brain (DLPFC[^3]) |  | 484 | 484 |  | [syn12299750](https://www.synapse.org/#!Synapse:syn12299750) | [phs000979.v2.p2](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000979.v2.p2) | [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/BrainSeq_QC_report.html)
[ROSMAP](https://doi.org/10.1038/nn.4632) | brain (DLPFC[^3]) |  | 576 | 576 |  | [syn3219045](https://www.synapse.org/#!Synapse:syn3219045) | [syn3219045](https://www.synapse.org/#!Synapse:syn3219045) | [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/ROSMAP_QC_report.html)
[Lepik_2017](https://doi.org/10.1371/journal.pcbi.1005766) | blood |  | 491 | 491 |  | Not deposited | Not deposited | [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/Lepik_2017_QC_report.html)
[FUSION](https://doi.org/10.1073/pnas.1814263116) | adipose, muscle |  | 559 | 302 |  | [phs001048.v2.p1](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001048.v2.p1) | [phs001048.v2.p1](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001048.v2.p1) | [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/FUSION_QC_report.html)
[GTEx](https://doi.org/10.1126/science.aaz1776) (v8) | 49 tissues |  | 15178 | 838 |  | [phs000424.v8.p2](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000424.v8.p2) | [phs000424.v8.p2](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000424.v8.p2) | 
[CAP](https://doi.org/10.1186/s12864-020-06966-4) | LCLs[^2] | statin | 296 | 148 |  | [phs000481.v3.p2](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000481.v3.p2) | [phs000481.v3.p2](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000481.v3.p2) | [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/CAP_QC_report.html)
[Peng_2018](https://doi.org/10.1371/journal.pgen.1007799) | placenta |  | 149 | 149 |  | [phs001586.v1.p1](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001586.v1.p1) | [phs001586.v1.p1](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001586.v1.p1) | [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/Peng_2018_QC_report.html)
[PhLiPS](https://doi.org/10.1016/j.stem.2017.03.017) | iPSCs[^1], hepatocytes |  | 168 | 87 |  | [phs001341.v1.p1](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001341.v1.p1) | [phs001341.v1.p1](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001341.v1.p1) | [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/PhLiPS_QC_report.html)
[CommonMind](https://doi.org/10.1038/s41597-019-0183-6) | brain (DLPFC[^3]) |  | 590 | 590 |  | [syn2759792](https://www.synapse.org/#!Synapse:syn2759792) | [syn2759792](https://www.synapse.org/#!Synapse:syn2759792) | [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/CommonMind_QC_report.html)
[Braineac2](https://doi.org/10.1038/s41467-020-14483-x) | brain (putamen, substantia nigra) |  | 167 | 110 |  | Not deposited | Not deposited | [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/Braineac2_QC_report.html)

### Microarray studies

| Study | Cell types or tissues | Conditions | Samples | Donors | Expression Atlas | Raw expression data | Raw genotype data | QC report |
|---|---|---|---|---|---|---|---|---|
[CEDAR](http://dx.doi.org/10.1038/s41467-018-04365-8) | CD4+ and CD8+ T cells, monocytes, neutrophils, platelet, B cells, ileum, rectum, transverse colon |  | 2388 | 322 |  | [E-MTAB-6667](https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6667) | [E-MTAB-6666](https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6666)| [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/CEDAR_QC_report.html)
[Fairfax_2012](http://dx.doi.org/10.1038/ng.2205) | B cells |  | 282 | 282 |  | [E-MTAB-945](https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-945) | [EGAD00010000144](https://www.ebi.ac.uk/ega/datasets/EGAD00010000144); [EGAD00010000520](https://www.ebi.ac.uk/ega/datasets/EGAD00010000520) | [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/Fairfax_2012_QC_report.html)
[Fairfax_2014](http://dx.doi.org/10.1126/science.1246949) | monocytes | IFN24, LPS2, LPS24 | 1372 | 424 | [E-MTAB-2232](https://www.ebi.ac.uk/gxa/experiments/E-MTAB-2232) | [E-MTAB-2232](https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2232) | [EGAD00010000144](https://www.ebi.ac.uk/ega/datasets/EGAD00010000144); [EGAD00010000520](https://www.ebi.ac.uk/ega/datasets/EGAD00010000520) | [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/Fairfax_2014_QC_report.html)
[Kasela_2017](http://dx.doi.org/10.1371/journal.pgen.1006643) | CD4+ and CD8+ T cells |  | 553 | 297 |  | [GSE78840](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE78840) | Not deposited| [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/Kasela_2017_QC_report.html)
[Naranbhai_2015](http://dx.doi.org/10.1038/ncomms8545) | neutrophils |  | 93 | 93 |  | [E-MTAB-3536](https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-3536) | [EGAD00010000144](https://www.ebi.ac.uk/ega/datasets/EGAD00010000144); [EGAD00010000520](https://www.ebi.ac.uk/ega/datasets/EGAD00010000520) |  [link]({{ site.domain }}{{ site.baseurl }}/static/QC_reports/Naranbhai_2015_QC_report.html)

### Imported studies
Although we generally prefer to reprocess all studies incldued in the eQTL Catalogue, we have also made original summary statistics available from a small number large-scale eQTL studies. These summary statistis can be accessed using both our REST API and FTP and follow the same general format. However, these studies have used independent gene expression quantification and genotype quality control workflows, so some systematic differences in eQTL results are expected.

| Study | Cell types or tissues | Conditions | Samples | Donors | Quantification | Raw expression data | Raw genotype data | Summary statistics source |
|---|---|---|---|---|---|---|---|---|---|
[GTEx_v8](http://dx.doi.org/10.1038/s41467-018-04365-8) | 49 tissues |  | 17382 | 948 | gene expression | [phs000424.v8.p2](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000424.v8.p2) | [phs000424.v8.p2](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000424.v8.p2) | [Google Cloud](gs://gtex-resources/GTEx_Analysis_v8_QTLs/GTEx_Analysis_v8_eQTL_all_associations) |

[^1]: induced pluripotent stem cells
[^2]: lymphoblastoid cell lines
[^3]: dorsolateral prefrontal cortex

## Acknowledgements
If you find the eQTL Catalogue useful in your research, please cite the eQTL Catalogue [**flagship paper**](https://doi.org/10.1101/2020.01.29.924266) as well as the orginal publications producing the raw data and aknowledge the funding that they received. 

#### BLUEPRINT
**Publication**: Chen, Lu, et al. "Genetic drivers of epigenetic and transcriptional variation in human immune cells." Cell 167.5 (2016): 1398-1414.

**Funding statement**: "This study makes use of data generated by the Blueprint Consortium. A full list of the investigators who contributed to the generation of the data is available from www.blueprint-epigenome.eu. Funding for the project was provided by the European Union's Seventh Framework Programme (FP7/2007- 2013) under grant agreement no 282510 - BLUEPRINT."

#### Fairfax_2012, Fairfax_2014, Naranbhai_2015
**Publications**:
- Fairfax, Benjamin P., et al. "Innate immune activity conditions the effect of regulatory variants upon monocyte gene expression." Science 343.6175 (2014): 1246949.
- Fairfax, Benjamin P., et al. "Genetics of gene expression in primary immune cells identifies cell type–specific master regulators and roles of HLA alleles." Nature genetics 44.5 (2012): 502.
- Naranbhai, Vivek, et al. "Genomic modulators of gene expression in human neutrophils." Nature communications 6 (2015): 7545.

**Funding statement**: "Funding for the project was provided by the Wellcome Trust under awards Grants 088891 [B.P.F.], 074318 [J.C.K.] and 075491/Z/04 to the core facilities at the Wellcome Trust Centre for Human Genetics), the European Research Council under the European Union’s Seventh Framework Programme (FP7/2007-2013) (281824 to J.C.K.), the Medical Research Council (98082, J.C.K.) and the National Institute for Health Research (NIHR) Oxford Biomedical Research Centre"

#### TwinsUK
**Publication**: "Buil, Alfonso, et al. "Gene-gene and gene-environment interactions detected by transcriptome sequence analysis in twins." Nature genetics 47.1 (2015): 88."

**Funding statement**: "TwinsUK is funded by the Wellcome Trust, Medical Research Council, European Union, the National Institute for Health Research (NIHR)-funded BioResource, Clinical Research Facility and Biomedical Research Centre based at Guy’s and St Thomas’ NHS Foundation Trust in partnership with King’s College London."

#### BrainSeq
**Publication**: "Jaffe, Andrew E., et al. "Developmental and genetic regulation of the human cortex transcriptome illuminate schizophrenia pathogenesis." Nat Neurosci 21.8 (2018): 1117-1125."

**Funding statement:** "This research was supported by the Intramural Research Program of the NIMH(NCT00001260, 900142)."

#### Schmiedel_2018
**Publication**: "Schmiedel, Benjamin J., et al. "Impact of genetic polymorphisms on human immune cell gene expression." Cell 175.6 (2018): 1701-1715."

**Funding statement:** "This work was funded by the William K. Bowes Jr Foundation (P.V.) and NIH grants R24AI108564 (P.V., B.P., A.R., M.K.), S10RR027366 (BD FACSAria II), and S10OD016262 (Illumina HiSeq 2500).

#### ROSMAP
**Publication:** "Ng, Bernard, et al. "An xQTL map integrates the genetic architecture of the human brain's transcriptome and epigenome." Nature neuroscience 20.10 (2017): 1418."

**Funding statement:** "Study data were provided by the Rush Alzheimer s Disease Center, Rush University Medical Center, Chicago. Data collection was supported through funding by NIA grants P304G10161, R014G15819, R014G17917, R01AG30!46, R014G36836, U014G32984, U014G46152, the Illinois Deparhnenr of Public Health, and the Translational Genomics Research Institute."

#### GENCORD
**Publication:** "Gutierrez-Arcelus, Maria, et al. "Passive and active DNA methylation and the interplay with genetic variation in gene regulation." elife 2 (2013): e00523."

**Funding statement:** Emmanouil T Dermitzakis was supported by grants from the European Research Council (260927), Swiss National Science Foundation (31003A_130342, CRSI33_130326) Louis-Jeantet Foundation, and the Blueprint Consortium. Stylianos E Antonarakis was supported by grants from the European Research Council (249968), Swiss National Science Foundation (144082), and the Blueprint Consortium.

#### van_de_Bunt_2015
**Publication:** "van de Bunt, Martijn, et al. "Transcript expression data from human islets links regulatory signals from genome-wide association studies for type 2 diabetes and glycemic traits to their downstream effectors." PLoS genetics 11.12 (2015)."

**Funding statement:** MvdB is supported by a Novo Nordisk postdoctoral fellowship run in partnership with the University of Oxford. ALG is a Wellcome Trust Senior Research Fellow in Basic Biomedical Science (095010/Z/10/Z). MIM is a Wellcome Trust Senior Investigator (WT098381) and a National Institute of Health Research Senior Investigator. PEM holds the Canada Research Chair in Islet Biology. This work was supported in part in Oxford, UK, by grants from the Medical Research Council (MRC; MR/L020149/1) and National Institutes of Health (NIH; R01 MH090941), and in Edmonton, Canada, by operating grants to PEM from the Canadian Institutes of Health Research (CIHR; MOP244739) and the ADI/Johnson & Johnson Diabetes Research Fund. Human islet isolations at the Alberta Diabetes Institute IsletCore were funded by the Alberta Diabetes Foundation and the University of Alberta. The National Institute for Health Research, Oxford Biomedical Research Centre funded islet provision at the Oxford Human Islet Isolation facility. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.